migrations/versions/db6b2ef8100f_.py

```diff
@@ -0,0 +1,35 @@
+"""empty message
+
+Revision ID: db6b2ef8100f
+Revises: e3a64b4ca634
+Create Date: 2024-01-29 14:12:38.070014
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = 'db6b2ef8100f'
+down_revision = 'e3a64b4ca634'
```

Comment:
Please change this to d0e7144947d0, so we have no migration script merging.

Reply:
[b7bc726](https://github.com/privacyidea/privacyidea/pull/3852/commits/b7bc726f2279a85bad77918536eb77dbabbf0523)